### PR TITLE
Linear model visualization

### PR DESCRIPTION
**Pull request by MikeLippincott**
Linear modeling to find feature importance for plate2

### CODE REVIEW

File: 3.Linear_Modeling/1a.fit_linear_model_1beta.ipynb [hunk 1]
```diff
@@ -0,0 +1,272 @@
+{
```
Comment (axiomcura) [inline 3.Linear_Modeling/1a.fit_linear_model_1beta.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #17.    from typing import Union
</pre><p> </p><p>Try moving this at the top of where all the standard libraries are being imported. <code>typing</code> is part of the python's standard library   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (axiomcura) [inline 3.Linear_Modeling/1a.fit_linear_model_1beta.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #19.        X = df.loc[:, model_covariates]
</pre><p>I would recommend standardizing input feature values before training model. here's a [source](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: 3.Linear_Modeling/1b.fit_linear_model_2b.ipynb [hunk 2]
```diff
@@ -0,0 +1,286 @@
+{
```
Comment (axiomcura) [inline 3.Linear_Modeling/1b.fit_linear_model_2b.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #15.        # treatment_df = feature_df[["Metadata_inducer1", "Metadata_inducer1_concentration"]]
</pre><p>consider removing this if you're no longer using it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Comment by axiomcura** [comment]
Nice PR! 

Two major comments.
1. I would recommend providing more fine grain commenting in where lots of pandas dataframe wrangling is occurring. readers can easily get lost with all the manipulation that is happening in a single block. 
2. It seem that your feature values are not standardized when training your model. Consider standardizing your dataset before training. Here's a good [source](https://towardsai.net/p/data-science/how-when-and-why-should-you-normalize-standardize-rescale-your-data-3f083def38ff) explaining when standardization should or should not be applied

**Review by axiomcura — APPROVED** [review]

File: 3.Linear_Modeling/1a.fit_linear_model_1beta.ipynb (hunk 1 above)
Comment (MikeLippincott) [reply 3.Linear_Modeling/1a.fit_linear_model_1beta.ipynb:1]: <p>@axiomcura turns out the data was already standardized via pycytominer. Mean of 0 and stdev of 1</p>
